version/version_test.go

```diff
@@ -1,114 +0,0 @@
-// Copyright (c) HashiCorp, Inc.
```

Comment:
I think that this file can be maintained. It was simply some tests that I introduced to cover untested portions of the code and doesn't rely on the internal ui version path.